samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -160,6 +160,17 @@ Building and running
 
 .. _peripheral_uart_sample_activating_variants:
 
+Experimental Bluetooth Low Energy Remote Procedure Call interface
+=================================================================
+
+To build the sample with a Bluetooth Low Energy Remote Procedure Call :ref:`ble_rpc` interface, use the following command:
+
+.. code-block:: console
+
+   west build samples/bluetooth/peripheral_uart -b board_name --sysbuild -S nordic-rpc-host -- -DFILE_SUFFIX=bt_rpc
+
+On the |nRF5340DKnoref|, you can build the sample with Bluetooth Low Energy Remote Procedure Call interface with the :ref:`ipc_radio` image using the same command.
```

Comment:
I think this line is unnecessary, because the RPC requires multicore board anyway.

Reply:
Removed